mobile/src/main/AndroidManifest.xml

```diff
@@ -131,9 +131,31 @@
                 <action android:name="android.app.action.NEXT_ALARM_CLOCK_CHANGED" />
             </intent-filter>
             <intent-filter>
-                <action android:name="android.intent.action.LOCALE_CHANGED"/>
+                <action android:name="android.intent.action.LOCALE_CHANGED" />
             </intent-filter>
         </receiver>
+
+        <activity
```

Comment:
I tried to add the intent filter to BackgroundTasksManager, but it seems that the NFC intents can only start activities, no broadcast receivers:

> Intent to start an activity when a tag with NDEF payload is discovered. 

from https://developer.android.com/reference/android/nfc/NfcAdapter.html#ACTION_NDEF_DISCOVERED